schemas/manifest.fbs

```diff
@@ -1,25 +1,34 @@
 include "sst.fbs";
 
 table Uuid {
-   high: uint64;
-   low: uint64;
+    high: uint64;
+    low: uint64;
 }
 
-table DbParent {
-   // Path to the parent database from which this database was cloned
-   path: string (required);
+// Reference to an external database.
+table ExternalDb {
+    // Path to root of the external database
+    path: string (required);
 
-   // Parent checkpoint ID that this clone was derived from
-   checkpoint: Uuid (required);
+    // Externally owned Checkpoint ID we've used to create an initial state of cloned database.
+    source_checkpoint_id: Uuid (required);
+
+    // Checkpoint ID this database has placed on the external database that prevents referenced
+    // data files from being GC'd. Both final_checkpoint_id and source_checkpoint_id should resolve
+    // to the same manifest_id as long as they both still exist.
+    final_checkpoint_id: Uuid (required);
```

Comment:
Tracking source and final checkpoint separately is a notable change since last pass. It allows us to make creation of final checkpoint idempotent across retries.